src/core/class.template.php

```diff
@@ -7,7 +7,9 @@
 
 namespace leantime\core {
 
+    use JetBrains\PhpStorm\NoReturn;
```

Comment:
am I missing where this is getting used?

Reply:
Removed